drracket/drracket/private/language.rkt

```diff
@@ -932,35 +915,32 @@
        (string-constant cancel)))
     
     (define (browse-callback)
-      (let ([ftf (send filename-text-field get-value)])
-        (let-values ([(base name _)
-                      (if (path-string? ftf)
-                          (split-path ftf)
-                          (values (current-directory) "" #f))])
-          (let* ([mzscheme? (currently-mzscheme-binary?)]
-                 [mode (current-mode)]
-                 [filename 
-                  (put-executable/defaults
-                   dlg
-                   base
-                   name
-                   mode
-                   (not mzscheme?)
-                   (case mode
-                     [(launcher)
-                      (if mzscheme?
-                          (string-constant save-a-mzscheme-launcher)
-                          (string-constant save-a-mred-launcher))]
-                     [(stand-alone)
-                      (if mzscheme?
-                          (string-constant save-a-mzscheme-stand-alone-executable)
-                          (string-constant save-a-mred-stand-alone-executable))]
-                     [(distribution)
-                      (if mzscheme?
-                          (string-constant save-a-mzscheme-distribution)
-                          (string-constant save-a-mred-distribution))]))])
-            (when filename
-              (send filename-text-field set-value (path->string filename)))))))
+      (define ftf (send filename-text-field get-value))
+      (let-values ([(base name _)
+                    (if (path-string? ftf) (split-path ftf) (values (current-directory) "" #f))])
+        (let* ([mzscheme? (currently-mzscheme-binary?)]
+               [mode (current-mode)]
+               [filename (put-executable/defaults
```

Comment:
the only change here seems to be changing the line breaks; is that intentional?

Reply:
Yeah, it's the result of running `fmt` on the code after replacing it. Though `let-to-define` handling nested `let` forms in a single pass would have avoided this confusion.